packages/vendor/webpack.plugin.js

```diff
@@ -15,13 +15,7 @@ class WebpackExportForemanVendorPlugin {
    * copy vendor-dist files to the consumer output path
    */
   applyCopyFiles(compiler) {
-    new CopyWebpackPlugin([
-      {
-        from: path.join(__dirname, 'dist/*'),
-        flatten: true,
-        ignore: ['manifest.json'],
-      },
-    ]).apply(compiler);
+    new CopyWebpackPlugin(manifest.files).apply(compiler);
```

Comment:
`manifest.files` returns the data in a format that `CopyWebpackPlugin` expects? because the previous code didn't look like a file list

Reply:
The previous codes used a glob pattern to copy everything from the `dist` folder (now this approach would copy prod and dev as well).
Now I provide an array of absolute paths by reading the relevant manifest, this format supported by the `CopyWebpackPlugin` as well.